postman/setup_postman

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
this is maybe a nit but did you consider just putting these in `bin/`? I can see the argument for having a postman-specific folder since it's such a key usecase ... but maybe even `examples/postman` would be better?

Reply:
I was wondering about this too. It's my assumption that someone may want to use the postman generator (`postman/generate_collection`) for development, but these extra scripts are only used for demoing. I think we could move the `postman/generate_collection` to `bin/generate_postman_collection`, and move these demo scripts to `examples/postman`. Thoughts?